### ISSUE TITLE
Animation#then does not accept a rejection handler anymore

### ISSUE THREAD

**Issue by mikepianka** [mistic100/Photo-Sphere-Viewer#735]
When clicking on the virtual tour node, a warning `Animation#then does not accept a rejection handler anymore` is thrown.

https://jsfiddle.net/mistic100/y0svuLpt/

![image](https://user-images.githubusercontent.com/47948499/186700305-36cfee02-0689-4747-89e0-9afbfe2e49bc.png)

**Comment by mistic100** [comment]
Yes I see it too, what I don't understand is where is it coming from.

When putting a breakpoint inside the Animation class I can see that the `onRejected` is native code.

I'll just remove the warning, it's been a few versions already.
